x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java

```diff
@@ -61,6 +61,8 @@
 
     static final Setting<Integer> ENRICH_FETCH_SIZE_SETTING =
         Setting.intSetting("index.xpack.enrich.fetch_size", 10000, 1, 1000000, Setting.Property.NodeScope);
+    static final Setting<Integer> ENRICH_POLICY_MAX_CONCURRENT_RUNS =
+        Setting.intSetting("index.xpack.enrich.policy.max_concurrent_runs", 50, 1, Setting.Property.NodeScope);
```

Comment:
might be outside the scope of this PR, why prefix with "index." for a NodeScope property ?

Reply:
This was just following the same pattern as the `ENRICH_FETCH_SIZE_SETTING`. Should I drop the prefix from both?

Reply:
Yes and I think we should also drop the `xpack` bit from the name (so just `enrich.*`).